udisksd.patch

```diff
@@ -1,83 +0,0 @@
-diff --git a/src/udisksdaemon.c b/src/udisksdaemon.c
```

Comment:
For posterity: we don't require this patch anymore because...?

Reply:
Because we're not using udisksd anymore. Snapcraft.yaml still doesn't reflect that, but I figured I'd go ahead and clean up the patch and the udisks script